package.json

```diff
@@ -23,7 +23,7 @@
     "build": "node ./scripts/build.js",
     "prebuild": "node ./scripts/prebuild.js",
     "postbuild": "node ./scripts/postbuild.js",
-    "dev:docs": "npx chokidar-cli \"./scripts/*\" \"./templates/**/*\" -c \"node ./scripts/tokens-documentation.js\"",
+    "dev:docs": "npx nodemon \"./scripts/*\" \"./templates/**/*\" -c \"node ./scripts/tokens-documentation.js\"",
```

Comment:
This doesn't look like the correct syntax for using `nodemon`, have you confirmed it's correct and works? The name of the script (`dev:docs`) also suggests it's run locally, if that's the case then it might be possible to move to a devDependency.

https://www.npmjs.com/package/nodemon